src/parser/permit-generation-module.ts

```diff
@@ -158,12 +158,16 @@ export class PermitGenerationModule implements Module {
         return result;
       }
     }
-    
+
     // Get treasury github user id
     const octokit = getOctokitInstance();
-    const { data: treasuryGithubData } = await octokit.users.getByUsername({ username: process.env.PERMIT_TREASURY_GITHUB_USERNAME });
+    const { data: treasuryGithubData } = await octokit.users.getByUsername({
+      username: process.env.PERMIT_TREASURY_GITHUB_USERNAME,
```

Comment:
Make sure to load this in at the top of the file, and to throw an error if its missing instead of potentially having this be invalid when its needed. I generally always make a helper function to load all the environment variables right at runtime start, and throw a detailed error if its missing any.

Reply:
It's already handled [here](https://github.com/ubiquibot/conversation-rewards/blob/f75d842d18bea073aff3d19fe98f0fc83420ae1a/src/parser/permit-generation-module.ts#L150-L152)

Reply:
I usually make a typebox schema for env in a separate file which validates the env and exports it. This makes it easier as there is only one entry point and it fails at the start of the program

Reply:
I think this should be handled in a different PR as this was not changed to resolve this one.